### ISSUE TITLE
why video download not working

### ISSUE THREAD

**Issue by candrapersada** [subhra74/xdm#87]
why video download not working for xdm version 7.2.10
![image](https://user-images.githubusercontent.com/36833664/77971391-be757700-7318-11ea-8a66-98caac7f932c.png)
![image](https://user-images.githubusercontent.com/36833664/77971417-cdf4c000-7318-11ea-8b50-e0e641af9220.png)
![image](https://user-images.githubusercontent.com/36833664/77971493-0399a900-7319-11ea-94e9-9936e991bc30.png)

**Comment by subhra74** [comment]
How XDM was installed? is it from Jar file?

**Comment by candrapersada** [comment]
yes from jar file. but video download not working without components.
![image](https://user-images.githubusercontent.com/36833664/78000139-01a60900-735e-11ea-9022-ea942ce5487c.png)
![image](https://user-images.githubusercontent.com/36833664/78000087-ea671b80-735d-11ea-9d8d-cbeb1d270a7c.png)

**Comment by subhra74** [comment]
You have to copy ffmpeg and youtube-dl to $HOME/.xdman folder or to the same folder where XDM is installed

**Comment by candrapersada** [comment]
i want link ffmpeg and youtube-dl

**Comment by subhra74** [comment]
FFmpeg: https://ffmpeg.zeranoe.com/
Youtube DL: https://github.com/ytdl-org/youtube-dl/releases/

**Comment by candrapersada** [comment]
link ffmpeg for Linux
